lib/Conversion/TTIRToD2M/TTIRToD2M.cpp

```diff
@@ -404,6 +404,8 @@ class D2MNamedElementwiseRewriter final
       std::is_same_v<TileOp, d2m::TileLtzOp> ||
       std::is_same_v<TileOp, d2m::TileLezOp>;
 
+  static constexpr bool isSignOp = std::is_same_v<TileOp, d2m::TileSignOp>;
```

Comment:
If following LLVM naming conventions, this would be called something like `kIsSignOp`